### ISSUE TITLE
Wrong number of decimals for Mauritius Rupee (MUR) CurrencyData

### ISSUE THREAD

**Issue by RajinDas** [gwtproject/gwt#9209]
The currency decimals for Mauritius Rupee is 2, but in the CurrenyData got by calling CurrencyList.get().lookup(currencyCode) is giving the decimals places as 0. 

Checked the CurrecyData.properties and found that the decimal is provided as zero in that file, even after correcting that in that properties file, I am still getting the decimal points as zero itself

**Comment by tbroyer** [comment]
You'd have to make the change in **all** `CurrencyData_<locale>.properties` too.

We're currently still using CLDR 25, whereas the latest version is 28. We somehow need to update.
However, CLDR 28 [still has `digits="0"`](http://www.unicode.org/repos/cldr/tags/release-28/common/supplemental/supplementalData.xml) so we'd likely have to patch locally (we already do for a couple things IIRC), but you'd have to raise an issue at the Unicode Consortium about the erroneous data.

/cc @manolo @jtamplin

**Comment by tbroyer** [comment]
Oh, and BTW, yes, it doesn't seem to match ISO 4217 which apparently says there should be 2 digits after the decimal separator: https://en.wikipedia.org/wiki/ISO_4217

**Comment by jtamplin** [comment]
I don't think this is urgent enough to maintain our own fork.  Instead, file a ticket against CLDR, and then the next time we update GWT's data to the next CLDR it will get picked up.

**Comment by RajinDas** [comment]
filed a ticket against CLDR
[CLDR] 9052: Wrong number of decimals for Mauritius Rupee (MUR) http://unicode.org/cldr/trac/ticket/9052

**Comment by jtamplin** [comment]
Great, thanks.

**Comment by niloc132** [comment]
Closed upstream as "needs-more-info", marked closed here, we can migrate when fixed upstream.
